incubator-tools/cds_dataset_creator/cds_dataset_creator.ipynb

```diff
@@ -0,0 +1,461 @@
+{
```

Comment:
<p>General question - Is this notebook even needed since you can now label documents by folder when uploading?</p><p> </p><p><a href="https://cloud.google.com/document-ai/docs/workbench/build-custom-splitter-processor#import_data_with_batch_labeling" target="_blank">https://cloud.google.com/document-ai/docs/workbench/build-custom-splitter-processor#import_data_with_batch_labeling</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/document-ai-samples/pull/720/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Yes, within the processor, we have the capability to import documents with the same page into a single label. However, for creating a multi-document file (https://screenshot.googleplex.com/63QMnnB5EapAiT7), which combines various types of document types into a single PDF, this tool can be utilized to generate labeled JSONs of that nature. The customer's request was for the creation of a tool to develop a multi-document dataset, which could potentially enhance the model's accuracy. Please let me know if my understanding is incorrect. Thanks.